### ISSUE TITLE
SUGGESTION - add README files to EVERY folder

### ISSUE THREAD

**Issue by betsyrosalen** [betsyCC/OAREDA_Syntax_Library#3]
I highly recommend that we add a README.md file to EVERY folder to store instructions and explanations about the code for each task.  This is much easier to read and more prominently featured than adding comments to the code for things that do not necessarily refer directly to lines of code.  Thoughts?

**Comment by CBLITTMAN-OAREDA** [comment]
I agree that every folder should have a README file for general information about the contents of the folder. But I also think that code comments are essential for users to understand what the code is doing when it may not be obvious, especially to newer analysts.
